Tools/Testing/CMakeLists.txt

```
@@ -23,21 +26,24 @@ set(ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../..")
 
 # Attitude Manager
 set(AM_FOLDER "${ROOT_DIR}/AttitudeManager")
-set(AM_INC "{$AM_FOLDER}/Inc")
-set(AM_SRC "{$AM_FOLDER}/Src")
+set(AM_INC "${AM_FOLDER}/Inc")
+set(AM_SRC "${AM_FOLDER}/Src")
+set(FLIGHTMODES_INC "${AM_FOLDER}/FlightModes/Inc")
+set(FLIGHTMODES_SRC "${AM_FOLDER}/FlightModes/Src")
 
-include_directories(${AM_INC})
+include_directories(${AM_INC} ${FLIGHTMODES_INC} ${AM_FOLDER}/Tests)
```

There aren't any headers in the Tests folder, so no need to include it in the list of include directories
```suggestion
include_directories(${AM_INC} ${FLIGHTMODES_INC})
```